concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-159	159	Pfeffer	Pfeffer	pepper	1385	PEPPER
+Steinthal-1875-1565-160	160	Cacao	Cacao	cocoa	2442	COCOA BEAN
+Steinthal-1875-1565-161	161	Cocosnuss	Cocosnuss	coconut	970	COCONUT
+Steinthal-1875-1565-162	162	Beere	Beere	berry	904	BERRY
+Steinthal-1875-1565-163	163	Kern	Kern	pip	1762	STONE (OF FRUIT)
+Steinthal-1875-1565-164	164	Schale	Schale	peel		
+Steinthal-1875-1565-165	165	schälen	schälen	to peel	275	PEEL
+Steinthal-1875-1565-166	166	Hülse	Hülse	husk	2363	CHAFF (HUSK)
+Steinthal-1875-1565-167	167	Fleisch der Früchte	Fleisch der Früchte	fruit pulp		
+Steinthal-1875-1565-168	168	Gummi	Gummi	gum		
+Steinthal-1875-1565-169	169	Baumwolle	Baumwolle	cotton	1850	COTTON
+Steinthal-1875-1565-170	170	Indigo	Indigo	indigo		
+Steinthal-1875-1565-171	171	Tabak	Tabak	tobacco	974	TOBACCO
+Steinthal-1875-1565-172	172	Oel	Oel	oil	1484	OIL (ORGANIC SUBSTANCE)
+Steinthal-1875-1565-173	173	Zucker	Zucker	sugar	862	SUGAR
+Steinthal-1875-1565-174	174	Stock (zum Gehen)	Stock (zum Gehen)	walking cane	1296	WALKING STICK
+Steinthal-1875-1565-175	175	Thier	Thier	animal	619	ANIMAL
+Steinthal-1875-1565-176	176	Hund bellt, beisst, wedelt mit dem Schwanze.	Hund bellt.	Dog barks.		
+Steinthal-1875-1565-177	177	Hund bellt, beisst, wedelt mit dem Schwanze.	Hund beisst.	Dog bites.		
+Steinthal-1875-1565-178	178	Hund bellt, beisst, wedelt mit dem Schwanze.	Hund wedelt mit dem Schwanze.	Dog wags its tail.		
+Steinthal-1875-1565-179	179	Katze miaut.	Katze miaut.	Cat meows.		
+Steinthal-1875-1565-180	180	Bulle	Bulle	bull	1008	BULL
+Steinthal-1875-1565-181	181	Kuh	Kuh	cow	1007	COW
+Steinthal-1875-1565-182	182	Ochs käut wieder.	Ochs käut wieder.	Ox ruminates.		
+Steinthal-1875-1565-183	183	Kalb	Kalb	calf	1989	CALF
+Steinthal-1875-1565-184	184	kalben	kalben	to calve		
+Steinthal-1875-1565-185	185	Stier brüllt.	Stier brüllt.	Bull roars.		
+Steinthal-1875-1565-186	186	Rindvieh	Rindvieh	bovine	3044	BOVINE
+Steinthal-1875-1565-187	187	Ziege mäckert.	Ziege mäckert.	Goat bleats.		
+Steinthal-1875-1565-188	188	Ziegenbock	Ziegenbock	He-goat	2475	MALE GOAT
+Steinthal-1875-1565-189	189	Schaf blökt.	Schaf blökt.	Sheep bleats.		
+Steinthal-1875-1565-190	190	Mutterschaf	Mutterschaf	ewe	1345	EWE
+Steinthal-1875-1565-191	191	Widder	Widder	ram	1344	RAM
+Steinthal-1875-1565-192	192	Hammel	Hammel	wether		
+Steinthal-1875-1565-193	193	Antilope	Antilope	antelope	3674	ANTELOPE
+Steinthal-1875-1565-194	194	Giraffe	Giraffe	giraffe	3089	GIRAFFE
+Steinthal-1875-1565-195	195	Hirsch	Hirsch	stag	1936	DEER
+Steinthal-1875-1565-196	196	Reh	Reh	deer	1936	DEER
+Steinthal-1875-1565-197	197	Schwein grunzt.	Schwein grunzt.	Pig grunts.		
+Steinthal-1875-1565-198	198	Pferd wiehert.	Pferd wiehert.	Horse neighs.		
+Steinthal-1875-1565-199	199	Esel ist träge.	Esel ist träge.	Donkey is idle.		
+Steinthal-1875-1565-200	200	Maulthier	Maulthier	mule (female horse, male donkey)	890	MULE
+Steinthal-1875-1565-201	201	Maulesel	Maulesel	mule (male horse, female donkey)	890	MULE
```

Comment:
Double mapping

Reply:
Oh, I totally missed the definition of MULE. Will unmap Maulesel (and change the translation, too).